packages/replayio/src/utils/installation/installLatestRelease.ts

```diff
@@ -44,14 +44,17 @@ export async function installLatestRelease() {
 
   const latestRelease = await getLatestRelease();
   const latestBuildId = latestRelease.buildId;
+  const latestVersion = latestRelease.version;
 
   // Write version metadata to disk so we can compare against the latest release and prompt to update
   debug("Saving release metadata to %s", metadataPath);
   writeToCache<MetadataJSON>(metadataPath, {
     chromium: {
       buildId: latestBuildId,
       installDate: new Date().toISOString(),
+      nativeVersion: latestVersion,
     },
+    node: undefined,
```

Comment:
We're mostly only supporting Chromium for now but I don't see the harm in thinking a little bit ahead for other runtimes.

Reply:
sure but this will get lost immediately anyway :D `JSON.stringify` will strip it away

Reply:
Yup! TS complains without it though bc of the metadata type.

Reply:
Maybe if I wrap a `Partial<...>` around the `Record<...>` it wouldn't?